main.go

```diff
@@ -35,7 +36,9 @@ var (
 
 func main() {
 	log.Namespace = "dp-frontend-router"
-
+	i :=0
```

Comment:
was this i var and the new `fmt.Println(i)` added for debugging purposes? If this is the case I think they should be removed